src/edu/psu/sweng500/team8/gui/SudokuGUI.java

```diff
@@ -66,10 +80,22 @@ public void cellChanged(Cell cell, int newNumber) {
 		clearMessage();
 		this.gameBoard.clearHighlightedIncorrectCells();
 		
+		this.board = this.gameSession.getGameBoard();
+		
+		
+		if (!board.hasOpenCells()) {
```

Comment:
That's what I'm saying -- it was old code that was refactored into gameIsComplete(). It must have been re-added during the merge. The fact that it's showing as a diff in this review means your branch is re-adding it. Please remove it before merging this.